src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -223,6 +258,13 @@ const customHTMLElementModels = {
     edited: defaultHTMLElementModels.span.extend({
         tagName: 'edited',
     }),
+    'muted-text': defaultHTMLElementModels.div.extend({
+        tagName: 'muted-text',
+        mixedUAStyles: styles.mutedTextLabel,
+    }),
+    comment: defaultHTMLElementModels.div.extend({
+        tagName: 'comment',
```

Comment:
Do we already use this tag on chat comments?

Reply:
In this PR I'm wrapping the comment's html in:

```
<RenderHTML
    html={`<comment>${fragment.html + (fragment.isEdited ? '<edited></edited>' : '')}</comment>`}
/>
```